apps/webClient/src/components/View/index.tsx

```diff
@@ -1,54 +1,90 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import clsx from "clsx";
+import { useLocation } from "react-router-dom";
 import { makeStyles } from "@material-ui/styles";
 import Container from "@material-ui/core/Container";
-import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import Logo from "../Logo";
 
 interface Props {
   children: React.ReactNode;
 }
 
-// placholder, should dynamically change via useLocation
-const LIGHT_BLUE = "rgb(177,32,41)";
-const GRADIENT =
+const RED = "rgb(177,32,41)";
+const RED_GRADIENT =
   "linear-gradient(45deg, rgba(177,32,41,1) 0%, rgba(244,121,32,1) 100%)";
+const GREEN = "rgb(14,124,63)";
+const GREEN_GRADIENT =
+  "linear-gradient(45deg, rgba(14,124,63,1) 0%, rgba(190,198,64,1) 100%)";
+
+const BLUE = "rgb(28,85,119)";
+const BLUE_GRADIENT =
+  "linear-gradient(45deg, rgba(28,85,119,1) 0%, rgba(46,192,209,1) 100%)";
+const YELLOW = "rgb(190,121,42)";
+const YELLOW_GRADIENT =
+  "linear-gradient(45deg, rgba(190,121,42,1) 0%, rgba(254,228,104,1) 100%)";
 
 const useStyles = makeStyles(theme => ({
   root: {
     width: "100vw",
-    height: "100vh",
+    height: "100%",
+    minHeight: "100vh",
     zIndex: 1,
     overflow: "hidden",
     display: "flex",
     flexDirection: "column",
   },
   main: {
     width: "100%",
+    height: "100%",
     flexGrow: 1,
-    overflowY: "auto",
-    overflowX: "scroll",
+    padding: 0,
+    marginTop: 128,
```

Comment:
Apologies for the mixup here.  The original styles were from an application where there was a sticky header at the top, which we don't have in this application.

I think we can remove `height` from the main class as `flexGrow` will ensure the content fills the screen.

Also just kind of curious what the padding: 0 achieves.

Finally, use `theme.spacing(x)` instead of `128` -- by doing this we can update padding across the entire site by adjusting the theme.  Think gmail's new "compact" setting.

Reply:
@JamesKemp296 -- after further thought, let's update this in a follow up PR.  Everything else looks good and there's no reason not to merge.

I am very picky about how the "top" few DOM elements are structured as its best not to mess with them once the foundation is set.